utils/dwdparse/dwdparse_test.go

```diff
@@ -434,3 +434,19 @@ func TestKeyRegexp(t *testing.T) {
 
 	test(t, rules, tests)
 }
+
+func TestKeyMalformedRegexp(t *testing.T) {
+	rules := []DWDirectiveRuleSpec{{
+		Command: "regexp",
+		RuleDefs: []DWDirectiveRuleDef{{
+			Key:  `*`, // missing argument to repetition operator '*'
```

Comment:
what happens if the key is empty?

Reply:
That looks like an existing bug. I'll fix it in another PR